schemas/audiences/segment.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/audiences/segment",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Assignment",
+  "type": "object",
+  "description":
+    "",
+  "definitions": {
+    "segment": {
+      "properties": {
+        "@id": {
+          "title": "Identity",
+          "type": "string",
+          "format": "uri",
+          "description": "Unique identity of the segment."
+        },
+        "xdm:endUserIDs": {
+          "title": "End User IDs",
+          "$ref": "https://ns.adobe.com/xdm/context/enduserids",
+          "description":
+            "Condensed, normalized encapsulation of an end user identifier that the segment is assigned to."
+        },
+        "xdm:name": {
+          "title": "Name",
+          "type": "string",
+          "description": "The common name associated with the segment."
+        },
+        "xdm:assignedOn": {
+          "title": "Assigned On",
+          "type": "string",
+          "format": "date-time",
+          "description": "The date the segment was assigned to the profile."
+        },
+        "xdm:validUntil": {
+          "title": "Valid Until",
+          "type": "string",
+          "format": "date-time",
+          "description": "The date the segment assignment is valid until."
+        },
+        "xdm:assignedBy": {
```

Comment:
I think we should use the properties we are using elsewhere
- `xdm:repositoryLastModifiedBy`
- `repo:lastModifiedDate`

If the assignment has been made by a program, let's link to "Namespace"